src/main/resources/com/veertu/plugin/anka/AnkaCloudSlaveTemplate/config.jelly

```diff
@@ -62,13 +50,24 @@
             </table>
         </f:entry>
 
+        <f:entry title="Maximum Allowed Nodes/Agents" description="The number of Jenkins Nodes/Agents this template and label is allowed to start (0 is unlimited)" field="instanceCapacity">
+            <f:textbox clazz="required number" default="0" />
+        </f:entry>
 
-        <f:entry title="${%Slave name template}" field="nameTemplate" description="Name template for a created slave">
-            <f:textbox default="" />
+        <f:entry title="${%Allowed Executors}" field="numberOfExecutors" description="This is the number of jobs/builds which can run on the started Jenkins Node/Agent that uses this Template">
+            <f:textbox clazz="required number" default="1" />
+        </f:entry>
+
+        <f:entry title="${%Anka VM Workspace Path}" field="remoteFS" description="Where Jenkins will store the files/workspace on the started Node/Agent">
+            <f:textbox clazz="required" default="/Users/anka/"/>
+        </f:entry>
+
+        <f:entry title="${%Keep Alive on Error}" field="keepAliveOnError" description="Keep the Node/Agent alive if the job running on it fails">
```

Comment:
Maybe add in the description that the VM stays alive as well, to make it clear. Not sure. What do you think?